connect-inject/container_init_test.go

```diff
@@ -158,6 +162,122 @@ services {
 }
 `,
 			"",
+			Handler{},
+			"",
+		},
+
+		// TODO do we need all of these tests, if the handler tests the enableMetrics and prometheusScrapePort methods?
```

Comment:
There are some common validation methods in subcommand/common, and connect-inject/handler. I think the edge case testing should be handled there, and here it should just be "happy path" cases of enabling metrics and configuring the prometheus scrape port and seeing they are configured.